schemas/com.mbta.ocs.tmov_light_rail.v1.json

```diff
@@ -0,0 +1,108 @@
+    },
+    "time": {
+      "$ref": "cloudevents.json#/properties/time"
+    },
+    "partitionkey": {
+      "type": "string",
+      "minLength": 1,
+      "description": "Key used to balance messages across multiple shards"
+    },
+    "data": {
+      "description": "The event payload.",
+      "$ref": "#/$defs/datadef"
+    }
+  },
+  "required": [
+    "id",
+    "source",
+    "specversion",
+    "type",
+    "data",
+    "time",
+    "partitionkey"
+  ],
+  "$defs": {
+    "datadef": {
+      "type": "object",
+      "allOf": [
+        { "$ref": "ocs-events-common.json#/$defs/tmovCommonBase" },
+        { "$ref": "ocs-events-common.json#/$defs/tmovCommonAdditional" },
+        {
+          "required": [
+            "routeNumber",
+            "lastAvi",
+            "dataSource",
+            "consist",
+            "speed",
+            "heading"
+          ],
+          "properties": {
+            "routeNumber": {
+              "description": "",
+              "type": "string"
+            },
+            "lastAvi": {
+              "description": "",
+              "type": "string"
+            },
+            "dataSource": {
+              "description": "The comm type from which positional data was obtained. `avi` indicates information was obtained from an AVI read via a wayside device. `nvl` (\"non-vital logic\") indicates information was obtained from a track circuit. `gps` indicates information was obtained from a GPS",
+              "$comment": "TODO: (1) Is it safe for this to be an enum? (2) Is it best to drop the `ci` suffix as RTR does?",
+              "type": "string",
+              "enum": ["avi", "nvl", "gps"]
+            },
+            "consist": {
+              "description": "Full consist, ie car numbers of this vehicle",
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "speed": {
+              "description": "Speed of vehicle, if dataSource is gps",
+              "$comment": "TODO: RTR uses an explicit 'unreported' value here. Should we just be optional / nullable?",
+              "oneOf": [
+                {
+                  "type": "number"
+                },
+                {
+                  "const": "unreported"
+                }
+              ]
+            },
+            "heading": {
+              "description": "Direction the vehicle is pointing, measured in degrees from Magnetic North. Only provided with GPS data source.",
+              "$comment": "TODO: RTR uses an explicit 'unreported' value here. Should we just be optional / nullable?",
+              "type": ["number", "null"]
+            },
+            "orientation": {
+              "description": "Whether the 'rear' or 'front' of each train is facing forward for each train in the consist. B means backward, A means forward. Only provided with AVI data source.",
+              "type": ["string", "null"]
+            }
```

Comment:
Took another pass at this and tried to make it align with the RTR lightrail messages. If I'm understanding it, that means each individual character in the string represents the orientation of a train car, as opposed to, say, having this be an array with "AB" vs "BA" values, like RTR has internally. E.g. `"AAB"` instead of `[ "AB, "AB, "BA" ]`.